pkgs/development/libraries/science/math/or-tools/default.nix

```diff
@@ -1,30 +1,74 @@
-{ lib, stdenv, fetchFromGitHub, cmake, abseil-cpp, gflags, which
-, lsb-release, glog, protobuf, cbc, zlib
-, ensureNewerSourcesForZipFilesHook, python, swig }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, abseil-cpp
+, bzip2
+, zlib
+, gflags
```

Comment:
can remove gflags and glog
see: https://github.com/google/or-tools/releases/tag/v8.1